src/site/_includes/js/eligibility.js

```diff
@@ -176,6 +176,18 @@ const cnst = {
       121650,
     ],
   },
+  hudvash: {
```

Comment:
Updated in [6ea421a](https://github.com/theunitedeffort/theunitedeffort.org/pull/629/commits/6ea421af31bf408192baca9fd00c69829bbdfa8c)

Reply:
That link does list income limits, but it's a secondary source.  It would be better to go directly to the primary source.  The primary source listed below that chart image is https://www.hcd.ca.gov/sites/default/files/docs/grants-and-funding/income-limits-2024.pdf

I actually just discovered that the chart numbers on that site do not even match the source document they linked to.  I do not know the reason for the discrepancy, but I suggest we go with the primary source.  

The limits in that sv@home page and the hcd.ca page are the HCD (California Department of Housing and Community Development) limits.  Since HUD-VASH seems to be under the HUD (U.S. Department of Housing and Urban Development) we should go with the HUD limits.  They are sometimes different.   Here is a [link to the HUD limits](https://www.huduser.gov/portal/datasets/il/il2024/2024summary.odn?states=6.0&data=2024&inputname=METRO41940M41940*0608599999%2BSanta+Clara+County&stname=California&statefp=06&year=2024&selection_type=county) that you can put in along with [this link](https://www.federalregister.gov/d/2024-17957/p-55) that tells us it's the 80% AMI values we should pay attention to.